### PR TITLE
Update max destinations covered by unexpired sources and max attributions per rate limit window limits to be per-site

### PR DESCRIPTION
**Pull request by akashnadan**
Explainer update for #661

### CODE REVIEW

File: EVENT.md [hunk 1]
```diff
@@ -876,13 +876,13 @@ The longer the cooldown windows are, the harder it is to abuse the API and join
 identity. Ideally attribution thresholds should be low enough to avoid leaking too
 much information, with cooldown windows as long as practically possible.
 
-Note that splitting these limits by the reporting origin introduces a possible
-leak when multiple origins collude with each other. However, the alternative
-makes it very difficult to adopt the API if all reporting origins had to share a
+Note that splitting these limits by the reporting site introduces a possible
```
Comment (csharrison) [inline EVENT.md:879]: Need to update line 871 as well

File: EVENT.md [hunk 2]
```diff
@@ -936,11 +936,11 @@ tradeoffs for privacy and utility.
 Because this limit is per source site, it is possible for different reporting
 origin on a site to push the other attribution sources out of the browser. See
 the [denial of service](#denial-of-service) for more details. To prevent this
-attack, the browser should maintain these limits per reporting origin. This
+attack, the browser should maintain these limits per reporting site. This
 effectively limits the number of unique sites covered by unexpired sources from
 any one reporting origin.
```
Comment (csharrison) [inline EVENT.md:941]:
```suggestion
any one reporting site.
```

File: EVENT.md (hunk 1 above)
Comment (akashnadan) [reply EVENT.md:879]: updated

File: EVENT.md (hunk 2 above)
Comment (akashnadan) [reply EVENT.md]: updated

**Review by csharrison — APPROVED** [review]

**Pull request merged**